model/src/DA/Finance/OrderTests.daml

```diff
@@ -0,0 +1,132 @@
+{-# LANGUAGE ApplicativeDo #-}
```

Comment:
Let's remove this file, this was just playing around with authorization models.

Reply:
or rather, move it into the tests directory - it can still serve as a valid test around the asset model